.github/workflows/notify_team_new_comment.yml

```diff
@@ -17,6 +17,10 @@ jobs:
 
     runs-on: ubuntu-latest
     steps:
+      - name: Escape title double quotes
+        id: escape_title
+        run: echo "ISSUE_TITLE=${${{ github.event.issue.title }}//\"/\\\"}" >> "$GITHUB_OUTPUT"
```

Comment:
I tried to run this in my shell with an example string in place of `${{ github.event.issue.title }}`, but it wouldn't execute.

I think sed probably needs to be used here to do the substitution?

Reply:
Did you include the string directly? I used a variable when testing it (screenshot is in the PR description) so I wonder if it contributed to it working. Could you please paste here the problematic expression and the error you got?

Reply:
I can use sed if it's better, would like to try it out on whatever was breaking on your side

Reply:
Yes, I used a literal string. I think that may be the difference, and the Github Action will do the replacement directly.

So I think updating it so that you first assign `title=${{ github.event.issue.title }}` and then reference that in the replacement? Then it should work without sed, just doesn't seem to work with a string literal.

Reply:
Ah I see, yes I think that should work. Will update, thank you.

Reply:
Updated